mobile/src/libs/components/button/button.tsx

```diff
@@ -29,40 +30,49 @@ const Button: React.FC<Properties> = ({
   isRounded,
   iconName,
   isVisuallyCentered,
+  color,
 }) => {
+  const renderIcon = (): JSX.Element => {
+    return (
+      <View
+        style={[
+          isVisuallyCentered && styles.visuallyCenteredButton,
+          isRounded && styles.buttonRounded,
+        ]}
+      >
+        <Icon
+          name={iconName ?? 'play'}
```

Comment:
```suggestion
          name={iconName as iconName}
```
In that stage you already have non nullable iconName because of the boolean below